src/ptwt/_util.py

```diff
@@ -29,6 +31,15 @@ def __len__(self) -> int:
         return len(self.dec_lo)
 
 
+WaveletDetailTuple2d = tuple[torch.Tensor, torch.Tensor, torch.Tensor]
+WaveletDetailDict = dict[str, torch.Tensor]
+
+WaveletCoeffDetailTuple2d = tuple[
+    torch.Tensor, Unpack[tuple[WaveletDetailTuple2d, ...]]
+]
+WaveletCoeffDetailDict = tuple[torch.Tensor, Unpack[tuple[WaveletDetailDict, ...]]]
```

Comment:
This appears in the documentation, no? Should we perhaps call it 3D-wavelet details coeffs? After all, it's called Dict now, but it's actually a tuple with dicts. This is the structure we inherit from 'pywt'; perhaps dropping the dict-type from the name reduces the risk of confusion.

Reply:
I find the naming to be quite hard :D I agree, having the `Dict` / `Tuple2d` in the name invokes the idea that the object itself is of this type. However, I didn't come up with a good way to put the difference between both types into the names. I thought about adding the dimensions (i.e. `Tuple2d -> 2d`, `Dict -> Nd`) but we also use the dict version in the separable 2d case.

Reply:
Yes, I agree. Could we create an alias for the separable 2d case? Would having both a separable coeffs and an Nd coeffs type be a solution?

Reply:
I renamed
* `WaveletCoeffDetailDict` to `WaveletCoeffNd`
* `WaveletCoeffDetailTuple2d` to `WaveletCoeff2d`
and introduced
* `WaveletCoeff2dSeparable` as an alias for `WaveletCoeffNd`
but kept the names for the detail coefficient type aliases.
Do you think that works?

Reply:
Yes! I think it does.